src/ring_theory/jacobson.lean

```diff
@@ -465,140 +438,7 @@ begin
-  (I : ideal (polynomial R)) [hI : I.is_maximal] :
-  ((quotient.mk I).comp C : R →+* I.quotient).is_integral :=
-begin
-  let I' : ideal R := I.comap C,
-  haveI : I'.is_prime := comap_is_prime C I,
-  let i : R →+* I'.quotient := quotient.mk I',
-  let f : polynomial R →+* polynomial I'.quotient := polynomial.map_ring_hom (quotient.mk I'),
-  have hf : function.surjective f := map_surjective i quotient.mk_surjective,
-  let J : ideal (polynomial I'.quotient) := I.map f,
-  have hIJ : I = J.comap f := begin
-    rw comap_map_of_surjective f hf,
-    refine le_antisymm (le_sup_left_of_le le_rfl) (sup_le le_rfl _),
-    refine λ p hp, polynomial_mem_ideal_of_coeff_mem_ideal I p (λ n, _),
-    rw ← quotient.eq_zero_iff_mem,
-    rw [mem_comap, ideal.mem_bot, polynomial.ext_iff] at hp,
-    simpa using hp n,
-  end,
-  haveI : J.is_maximal := or.rec_on (map_eq_top_or_is_maximal_of_surjective f hf hI)
-    (λ h, absurd (trans (h ▸ hIJ : I = comap f ⊤) comap_top : I = ⊤) hI.1) id,
-  let i' : I.quotient →+* J.quotient := quotient_map J f le_comap_map,
-  have hi' : function.injective i' := quotient_map_injective' (le_of_eq hIJ.symm),
-  let ϕ : R →+* I.quotient := (quotient.mk I).comp C,
-  let ϕ' : I'.quotient →+* J.quotient := (quotient.mk J).comp C,
-  refine ring_hom.is_integral_tower_bot_of_is_integral ϕ i' hi' _,
-  refine (ring_hom.ext (λ _, by simp) : ϕ'.comp i = i'.comp ϕ) ▸ ring_hom.is_integral_trans i ϕ'
-    (i.is_integral_of_surjective quotient.mk_surjective)
-    (quotient_mk_comp_C_is_integral_of_jacobson' J (λ x hx, _)),
-  obtain ⟨z, rfl⟩ := quotient.mk_surjective x,
-  rwa [quotient.eq_zero_iff_mem, mem_comap, hIJ, mem_comap, coe_map_ring_hom, map_C],
-end
-
-lemma comp_C_integral_of_surjective_of_jacobson {R : Type*} [integral_domain R] [is_jacobson R]
-  {S : Type*} [field S] (f : (polynomial R) →+* S) (hf : function.surjective f) :
-  (f.comp C).is_integral :=
-begin
-  haveI : (f.ker).is_maximal := @comap_is_maximal_of_surjective _ _ _ _ f ⊥ hf bot_is_maximal,
-  let g : f.ker.quotient →+* S := ideal.quotient.lift f.ker f (λ _ h, h),
-  have hfg : (g.comp (quotient.mk f.ker)) = f := quotient.lift_comp_mk f.ker f _,
-  rw [← hfg, ring_hom.comp_assoc],
-  refine ring_hom.is_integral_trans _ g (quotient_mk_comp_C_is_integral_of_jacobson f.ker)
-    (g.is_integral_of_surjective (quotient.lift_surjective f.ker f _ hf)),
-end
-
-lemma is_maximal_comap_C_of_is_jacobson {R : Type*} [integral_domain R] [is_jacobson R]
-  (P : ideal (polynomial R)) [hP : P.is_maximal] : (P.comap (C : R →+* polynomial R)).is_maximal :=
-begin
-  have := is_maximal_comap_of_is_integral_of_is_maximal' _
-    (quotient_mk_comp_C_is_integral_of_jacobson P) ⊥ (by rwa bot_quotient_is_maximal_iff),
-  rwa [← comap_comap, ← ring_hom.ker_eq_comap_bot, mk_ker] at this,
-end
-
+end integral_domain
 end polynomial
 
-namespace mv_polynomial
-open mv_polynomial
-
-lemma is_jacobson_mv_polynomial_fin [H : is_jacobson R] :
```

Comment:
Are these proofs deliberately deleted? Is the intent to add them back before merging?

Reply:
I thought that I had not deleted anything in the pushed version: let me check!